docs/automated-workflows/dagman-resubmit-failed.rst

```diff
@@ -1,25 +1,22 @@
 Resubmitting a Failed DAG
 =========================
 
-When debugging a DAG in which something has gone wrong, a first
-determination is whether a resubmission will use a Rescue DAG or benefit
-from recovery. The existence of a Rescue DAG means that recovery would
-be inappropriate. A Rescue DAG is has a file name ending in
-``.rescue<XXX>``, where ``<XXX>`` is replaced by a 3-digit number.
+DAGMan has two ways of restarting a DAG that has failed, Rescue and Recovery.
+Rescue mode is most comman for resubmitting a DAG manually while recovery
+mode is most likely to occur automatically if a crash or something occurs.
 
-Determine if a DAG ever completed (independent of whether it was
-successful or not) by looking at the last lines of the ``.dagman.out``
-file. If there is a line similar to
+If a DAG has left the queue and the ``.dagman.out`` file doesn't end
+with a successful exit line similar to
 
 .. code-block:: text
 
     (condor_DAGMAN) pid 445 EXITING WITH STATUS 0
 
-then the DAG completed. This line explains that the :tool:`condor_dagman` job
-finished normally. If there is no line similar to this at the end of the
-``.dagman.out`` file, and output from :tool:`condor_q` shows that the
-:tool:`condor_dagman` job for the DAG being debugged is not in the queue,
-then recovery is indicated.
+Then the DAG has failed and needs to be restarted. Resubmission should
+be done via a Rescue DAG unless none exist, then the use of Recovery
```

Comment:
replace "Rescue DAG unless none exist, then the use of ..." with "Rescue DAG if the file exists, otherwise DAGMan will use Recover mode."